test/benchmark_test.cc

```diff
@@ -70,7 +68,7 @@ static void BM_FactorialRealTime(benchmark::State& state) {
   ss << fac_42;
   state.SetLabel(ss.str());
 }
-BENCHMARK(BM_FactorialRealTime);
+BENCHMARK(BM_FactorialRealTime)->UseRealTime();
```

Comment:
no need for this .. just add this macro to the BM_Factorial benchmark above to show how it works.